src/main/webapp/index.html

```diff
@@ -122,7 +122,7 @@ <h1>Favorite SDKs</h1>
         };
 
         // Initialize Firebase
-            firebase.initializeApp(firebaseConfig);
+        firebase.initializeApp(firebaseConfig);
```

Comment:
I see only this change (indentation fix) in the entire PR. Wondering if the PR was not created correctly. Did you want to add anything else here?

Reply:
I made all the changes requested in the Page Adjustment Pr. I'm not sure how else I was supposed to move the file to another pr

Reply:
Did you delete those changes in PageAddjustments PR? I cant see the diff if you dont...

Reply:
I deleted the index.html file on the PageAdjustments branch